data/webhid.json

```diff
@@ -1,12 +1,5 @@
 {
-  "url": "https://github.com/nondebug/webhid/blob/master/EXPLAINER.md",
-  "title": "WebHID",
```

Comment:
WebHID isn't in Specref yet, so title needs to be retained for now.
Group shouldn't be necessary though.

Reply:
I pushed a commit to add the spec title. Thanks!